start-from-zero/new-evaluation-zero.html

```diff
@@ -4,66 +4,82 @@
     </nav>
     
+    
+   
+    
     <section class="form-candidate">
         <div class="container">
            <div class="row">
-
-                <form action="" class="user-input">
-                    <div class="field field-new">
-                        <input id='input-zero' type="text" name='username' placeholder="Username">
-                        <input id='input-zero' type="password" name='password' placeholder="Password">
+                <div class="col-md-12">
+                    <form action="" class="user-input" method='post'>
+                    <div class="field-ev">
+                        <input id='input-username' type="text" name='username' placeholder="Username">
+                        <input id='input-password' type="password" name='password' placeholder="Password">
+                        <input id='input-calendar' type="date">
+                        
                     </div>
                     
                     
                     <h4>Technical Level</h4>
-                    <div class='blue-line2'>
-                        <div class='trainee-blue'>Trainee</div>
-                        <div class='junior-blue'>Junior</div>
-                        <div class='middle-blue'>Middle</div>
-                        <div class='senior-blue'>Senior</div>
-                    </div>
-                    <div class="field-radio">
-                       <div class='trainee-radio'>
-                           <input type="radio" name='junior'>
-                       </div>
-                       <div class="junior-radio">
-                           <input type="radio" name='junior'>
-                           <input type="radio" name='junior'>
-                           <input type="radio" name='junior'>
-                       </div>
-                       <div class="middle-radio">
-                            <input type="radio" name='junior'>
-                            <input type="radio" name='junior'>
-                            <input type="radio" name='junior'>
-                       </div>
-                       <div class="senior-radio">
-                            <input type="radio" name='junior'>
-                            <input type="radio" name='junior'>
-                            <input type="radio" name='junior'>
-                       </div>
+                    <table>
                         
-                    </div>
+                        <tbody class='tbody-new'>
+                            <tr class='tr-first'>
+                                <td>Trainee</td>
+                                <td>Junior</td>
+                                <td>Middle</td>
+                                <td>Senior</td>
+                            </tr>
+                            
+                            
+                            <tr class="tr-third">
+                                <td>
+                                    <input type="radio" name='junior'>
```

Comment:
A whole lotta radio buttons here, all sharing the same `name` attribute, but none with a `value` or corresponding `label`. How is a user going to know which one to clickity-click?